pyfolio/tests/test_pos.py

```diff
@@ -121,3 +125,41 @@ def test_get_turnover(self):
         result = get_turnover(transactions, positions, period='M')
         expected = Series([10.0], index=index)
         assert_series_equal(result, expected)
+
+    def test_sector_exposure(self):
+        """
+        Tests sector exposure mapping and rollup.
+
+        """
+        dates = date_range(start='2015-01-01', freq='D', periods=20)
+        positions = DataFrame([[1.0, 2.0, 3.0, 10.0]]*len(dates),
+                              columns=[0, 1, 2, 'cash'], index=dates)
+        mapping_d = {
```

Comment:
These test cases can be cleaner done with nose.parameterize (see some of the other tests for examples).